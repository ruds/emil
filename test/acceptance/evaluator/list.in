[];
[1, 2, 3, 4];
[(1, "foo"), (2, "bar"), (3, "baz")];
[] : int list;
[1, 2, 3, 4] : bigint list;
