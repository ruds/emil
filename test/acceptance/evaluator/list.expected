val it = nil : 'a list
val it = (::) (1, (::) (2, (::) (3, (::) (4, nil)))) : bigint list
val it = (::) ((1, "foo"), (::) ((2, "bar"), (::) ((3, "baz"), nil))) : (bigint * string) list
val it = nil : int list
val it = (::) (1, (::) (2, (::) (3, (::) (4, nil)))) : bigint list
